### DISCUSSION TITLE
Dashboard Issues on Formula(HTML) / Saved Search display

### DISCUSSION THREAD

**Discussion by tlchapman091** [burkybang/SuiteAdvanced-Feedback#142] [Q&A] [answered]
Hello, 

W/O SuiteAdvancded enabled my dashboard saved search looks like the following: 
![NativeNS](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/129122060/3ffd5619-350f-47fb-8c97-c073565dadfa)

With SuiteAdvanced enabled, my search looks like this: 
![Screenshot 2024-04-19 133605](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/129122060/9a684b98-a229-4d5b-b62a-9713a9aaef10)

Essentially turning my white text to blue.

**Comment by burkybang** [comment, marked as answer]
Just use !important in your style attribute.

Example:

```html
<a href="" style="color:white !important">FedEx Tracking</a>
```
